common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -220,10 +220,11 @@ class NativeAuthMsalController : BaseNativeAuthController() {
                 is SignInTokenApiResult.MFARequired, is SignInTokenApiResult.CodeIncorrect,
                 is SignInTokenApiResult.UserNotFound, is SignInTokenApiResult.InvalidCredentials,
                 is SignInTokenApiResult.UnknownError -> {
-                    Logger.warn(
+                    Logger.warnWithObject(
                         TAG,
                         tokenApiResult.correlationId,
-                        "Unexpected result: $tokenApiResult"
+                        "Unexpected result: ",
+                        tokenApiResult
```

Comment:
More of a question than a request: From what I can see, there is a correlation ID on most of the result objects, which will be logged from the `ILoggable` interface. With that, do we need the additional `tokenApiResult.correlationId` parameter explicitly passed on line 225 as well? Might make the logs cleaner. Seems to be the case for a few lines in this file.

Reply:
Good question. I haven't made any changes to other parts of logging, as that would make this PR bigger than it is, and @Yuki-YuXin  is working on that. My proposal would be to proceed with the version of this PR, and take a look at what you're pointing out in Yu's "log cleanup" work. Wdyt?

Reply:
Makes sense 👍